### PR TITLE
EO-231 

### PR DESCRIPTION
**Pull request by ischer**
engagement rate only shown when count_accepted present, not hidden if rate 0, fixed display of rate

### CODE REVIEW

File: src/helpers/abTesting.js [hunk 1]
```diff
@@ -59,3 +59,11 @@ export const calculateTotalSampleSize = ({ default_template, variants }) =>
     (sum, variant) => sum + parseInt(variant.sample_size, 10),
     parseInt(default_template.sample_size, 10)
   );
+
+/**
+ * Checks if a test has any variants which have been delivered to
+ */
+export const hasTestDelivered = (abTest) => {
+  const variants = [abTest.default_template].concat(abTest.variants);
```
Comment (bkemper) [inline src/helpers/abTesting.js]: Feel free to use spread here.  `const variants = [abTest.default_template, ...abTest.variants];`

File: src/helpers/abTesting.js [hunk 2]
```diff
@@ -59,3 +59,11 @@ export const calculateTotalSampleSize = ({ default_template, variants }) =>
...
+  return !!_.find(variants, (variant = {}) => variant.count_accepted > 0);
```
Comment (bkemper) [inline src/helpers/abTesting.js]: `Boolean` can be a nice replacement for `!!`.

File: src/helpers/abTesting.js (hunk 1 above)
Comment (ischer) [reply src/helpers/abTesting.js]: good call - definitely way better this way

File: src/helpers/abTesting.js (hunk 2 above)
Comment (ischer) [reply src/helpers/abTesting.js]: nice - yeah, much clearer.

**Pull request merged**